backends/rapidpro/backend_test.go

```diff
@@ -708,6 +708,9 @@ func (ts *BackendTestSuite) TestSentExternalIDCaching() {
 	// mimic a delay in that status being written by reverting the db changes
 	ts.b.db.MustExec(`UPDATE msgs_msg SET status = 'W', external_id = NULL WHERE id = 10000`)
 
+	// Add 1 second delay to make sure the status update have distict times
+	time.Sleep(time.Millisecond * 1000)
```

Comment:
I think move this line up to 703... this is so that the status update is written by the batcher no? And make it 600 ms